### PR TITLE
revamp user profile page to new ui

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- I worked on revamping user profile page to new ui
- fix the text display when no user found

image

image (10)

image

image

https://github.com/open-metadata/OpenMetadata/assets/66266464/46cd6b3b-95ce-4999-9efe-24eb6b4564d2

#
### Type of change:

- [] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Users/UsersProfile/user-profile.less [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@import url('../../../styles/variables.less');
+
+.user-profile-container {
+  padding: 30px 0;
+  height: 260px;
+  background: @user-profile-background;
+
+  .ant-col {
+    height: 100%;
+    overflow: hidden;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Users/UsersProfile/user-profile.less:23]: Let's review this CSS tomorrow

File: openmetadata-ui/src/main/resources/ui/src/components/common/Chip/Chip.component.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+  return (
+    <Space wrap data-testid="chip-container" size={4}>
+      {data.slice(0, USER_DATA_SIZE).map((item) => getChipElement(item))}
+      {hasMoreElement && (
+        <Popover
+          className="cursor-pointer"
+          content={
+            <Space wrap size={6}>
+              {data.slice(USER_DATA_SIZE).map((item) => getChipElement(item))}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/Chip/Chip.component.tsx:68]:
```suggestion
              {data.slice(USER_DATA_SIZE).map(getChipElement)}
```

File: openmetadata-ui/src/main/resources/ui/src/components/common/description/Description.tsx [hunk 3]
```diff
@@ -210,6 +210,7 @@ const Description: FC<DescriptionProps> = ({
             id="center">
             {description?.trim() ? (
               <RichTextEditorPreviewer
+                className={reduceDescription ? 'max-two-lines' : ''}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/description/Description.tsx:213]: We are moving to DescriptionV1, can you check if we do that

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 4]
```diff
@@ -172,6 +172,15 @@ a[href].link-text-grey,
   white-space: pre-wrap;
 }
 
+//  Reduce text to 2 lines
+.max-two-lines {
```
[the remark this answers is not in the document]
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:176]: yes, we have removed it. Previously it was used in `explore-search-card`.

File: openmetadata-ui/src/main/resources/ui/src/components/Users/UsersProfile/user-profile.less (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/Users/UsersProfile/user-profile.less:23]: we, don't need it, have removed this.

File: openmetadata-ui/src/main/resources/ui/src/components/Users/Users.style.less [hunk 5]
```diff
@@ -45,6 +46,10 @@
 }
 
 .user-layout {
+  .ant-col {
+    padding-top: 0;
+  }
```
[the remark this answers is not in the document]
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/Users/Users.style.less:51]: this is used to remove the padding from the page-layout component, and we use the class there to add it and we can't remove or add the class conditionally from the component just for only this case.

**Comment by Ashish8689** [comment]
@chirag-madlani Chanegs made as per comments and fix the cypress along with it!

**Pull request merged**